### DISCUSSION TITLE
Flutter Web PR Preview Channels

### DISCUSSION THREAD

**Discussion by markbreuss** [codemagic-ci-cd/codemagic-docs#2314] [Ideas]
### Use case

```Text
When reviewing PR's I'd like to be able to immediatelly click through the feature/bugfix.

There are already projects for this, but I'd like to use the codemagic build pipeline for that.
```

### Idea

```Text
Basically I'd like to have a flutter web url hosted on codemagic static pages for every pull request.
```

**Comment by markbreuss** [comment]
One possible way for that would be If I could use the PR / commit ID as a variable in the "deploy to codemagic static pages"

**Comment by mikhail-tokarev** [comment]
Thank you for your proposal and the opportunity to consider it. Unfortunetelly after careful consideration and internal discussions, we regret to inform you that we are unable to prioritise it at this moment. While we cannot move forward with this particular proposal, we value our relationship and hope to explore other opportunities together in the future.

**Reply by markbreuss** [reply]
Thanks for letting me know and considering- appreciate it!
